app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+  before_action :require_login, only: %i[create show]
+
+  def index
+    @event = Event.all
+    @attendance = Attendance.new
+    @past_events = Event.past_events
+    @upcoming_events = Event.upcoming_events
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.created_events.build(event_params)
+
+    if @event.save
+      redirect_to event_path(@event)
+
+    else
+      render :new
+    end
+  end
+
+  def show
+    @event = Event.find(params[:id])
+  end
```

Comment:
- [ ] You should list events created by the user, i.e. `@created_events = current_user.events` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)).
- [ ] list events previoulsy attendented by user **using your event model scope**, i.e. `@past_events = current_user.attended_events.previous` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
- [ ] list upcoming events previously attended by user **using your event model scope**, i.e. `@upcoming_events = current_user.attended_events.upcoming` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))